libretro/retro_events.c

```diff
@@ -148,15 +148,15 @@ t_button_cfg btnPAD[MAX_PADCFG] = {
    }},
    {{ // JOYSTICK PLAYER 1 CFG
    CPC_KEY_JOY_FIRE2,   // B
-   CPC_KEY_NULL,        // Y
+   CPC_KEY_JOY_FIRE1,   // Y
```

Comment:
Please remove this change. These buttons are used to make binding simple, see: #63